samples/RT-sample-app/nodejs-mutual-dependency/README.md

```diff
@@ -0,0 +1,145 @@
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+## Run the app (Manually Uploading to Teams)
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+> the Teams service needs to call into the bot.
+
+### 1. Setup for Bot
+- In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2)
+
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+
+- While registering the bot, use `https://<your_tunnel_domain>/api/messages` as the messaging endpoint.
+    > NOTE: When you create your bot you will create an App ID and App password - make sure you keep these for later.
+
+### 2. Setup NGROK
+1) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http 3978 --host-header="localhost:3978"
+    ```
+   Alternatively, you can also use the `dev tunnels`. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
+
+   ```bash
+   devtunnel host -p 3978 --allow-anonymous
+   ```
+
+### 3. Setup for code
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+2) In a terminal, navigate to `samples/RT-sample-app/nodejs-mutual-dependency`
+
+3) Install modules
+
+    ```bash
+    npm install
+    ```
+
+4) Update the `custom-environment-variables` configuration for the bot to use the `MicrosoftAppId` and `MicrosoftAppPassword`, `BaseUrl` with application base url.
+
+5) Update the `default` configuration for the bot to use the `appId` and `appPassword`.
+
+5) Run your app
+
+    ```bash
+    npm start
+    ```
+
+## RT Sample App Feature
+**This sample feature is to cover Mutual Dependency like Bot, Tab, ME depends on each other**
+1) Static Tab + ME + BOT => the three mutually depends on each other
+
+## Running the sample
+
+**Install App:**
+
+![InstallApp](Images/1.Install_App.png)
+
+**Mutual Dependency In Teams**
+
+![TeamsBot](Images/2.PersonalBot.png)
+
+![TeamsMsgExt](Images/3.PersonalSelectMsgExtAction.png)
+
+![TeamsMsgExt](Images/4.PersonalMsgExtAction.png)
+
+![TeamsActionSubmit](Images/5.PersonaMsgExtActionSubmit.png)
+
+![TeamsTab](Images/6.StaticTab.png)
+
+![TeamsTaskmodule](Images/7.TaskModule.png)
+
+**Outlook**
+
+**actionME dependsOn bots, actionME won't show up in Outlook**
```

Comment:
AFAIK, "actionME" isn't an official name we use publicly. We should use "action command" instead.

Reply:
fixed